### ISSUE TITLE
Transparency is not mentioned

### ISSUE THREAD

**Issue by fantasai** [w3c/AB-public#218]
One of our unstated principles is transparency: we "show our work", not just our work products. It enables more effective open participation, and supports thorough review and consensus. Might be useful to mention it somewhere.

**Comment by cwilso** [comment]
I agree this could be better captured.  I can see a few ways to do so:

1. Add transparency as a new principle.
2. Call out the need for transparency as a basis for consensus-building.
3. Incorporate transparency into the principle of open participation (as @fantasai said, it enables more effective participation).
4. Add this to the phrasing of collaboration; however, that's a bit more related to relationships with other SDOs.

I'm somewhat inclined, even though I would normally be disinclined to do so, to say #1: this is a separate principle, something along the lines of:

> **Transparency**: It is critically important that we demonstrate the reasoning behind our decisions. We show our work, documenting the decision making process in order to support better consensus, trust in our decisions, and encourage more open participation.

Thoughts?  I can write up a PR.

**Comment by frivoal** [comment]
I strongly support this addition in principle, and I think this first attempt at capturing it is reasonable.
